src/RGUnityBots/Packages/manifest.json

```diff
@@ -12,6 +12,7 @@
     "com.unity.timeline": "1.6.4",
     "com.unity.ugui": "1.0.0",
     "com.unity.visualscripting": "1.8.0",
+    "com.unity.nuget.mono-cecil": "1.11.4",
```

Comment:
This is just a Unity package that adds the Mono.Cecil assemblies. So, the only implication I can think of is a corner case where (1) the game project is already using Mono.Cecil, and (2) they have manually added the Mono.Cecil assemblies to their project directory rather than using the package provided by Unity. This would cause an assembly conflict, and I would suggest using the UPM package in this case.

Reply:
Also, to answer the question more broadly about the instrumentation: it happens behind the scenes as an added post-processing step to the build pipeline, and only makes any changes if legacy input API usage is detected. Any assemblies that do not use it will not be affected, and even for those that we do modify we preserve the debug information. So, from the developer's perspective no action is needed / everything should look the same.